frontend/packages/data-portal/app/components/Select.tsx

```diff
@@ -91,6 +111,9 @@ export function Select({
       </div>
 
       <InputDropdown
+        // For some reason input dropdown says `className` is not defined even though it is.
```

Comment:
I did see this https://github.com/chanzuckerberg/sci-components/blob/252c86cbffa7c38b901ac501be5ef72d9998bb45/packages/components/src/core/InputDropdown/style.ts#L49 though if it's just a temporary hack for them that might not be much better